agent/agent-bootstrap/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ dependencies {
     exclude("org.slf4j", "slf4j-simple")
   }
   implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api")
+  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api-semconv")
```

Comment:
what is semconv? please explain

Reply:
`instrumentation-api` was recently split out into 2 artifacts: `instrumentation-api` and `instrumentation-api-semconv`

Reply:
what does semconv stand for?

Reply:
semantic conventions